tests/storage/utils.py

```
@@ -536,3 +537,48 @@ def check_file_in_vm(
         vm_console.expect(pattern=file_content, timeout=TIMEOUT_20SEC)
+
+
+@contextmanager
+def vm_restore_with_prefix_policy(
+    name: str,
+    namespace: str,
+    vm_name: str,
+    snapshot_name: str,
+    client: DynamicClient,
+    prefix_policy: str,
+    dry_run: bool,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)